src/altinn-app-frontend/src/components/message/ErrorReport.tsx

```diff
@@ -80,22 +89,74 @@ const ErrorReport = ({ components }: IErrorReportProps) => {
         return;
       }
       ev.preventDefault();
-      if (currentView === error.layout) {
-        dispatch(
-          FormLayoutActions.updateFocus({
-            focusComponentId: error.componentId,
-          }),
-        );
-      } else {
+      if (currentView !== error.layout) {
         dispatch(
           FormLayoutActions.updateCurrentView({
             newView: error.layout,
             runValidations: null,
             returnToView: currentView,
-            focusComponentId: error.componentId,
           }),
         );
       }
+
+      const nodes = nodesInLayout(layouts[error.layout], repeatingGroups);
+      const componentNode = nodes.findById(error.componentId);
+
+      // Iterate over parent repeating groups
+      componentNode.parents().forEach((parent, i, allParents) => {
+        if (
+          parent.item?.type == 'Group' &&
+          parent.item.edit?.mode !== 'likert' &&
+          parent.item.maxCount > 1
+        ) {
+          const child =
+            i == 0
+              ? componentNode
+              : (allParents[i - 1] as AnyChildNode<'unresolved'>);
+
+          // Go to correct multiPage page if necessary
+          if (
+            (parent.item as RepeatingGroupHierarchy<'unresolved'>).edit
+              ?.multiPage
+          ) {
+            const childrenWithMultiPageIndex = (
+              layouts[error.layout].find(
+                (c) => c.id === parent.item.id,
```

Comment:
I see it's unfortunate that I intentionally removed the multipage stuff now.. :wink: Btw, `baseComponentId` might be better than `parent.item.id`, as it would fail if the parent was a nested repeating group.

Reply:
We could possibly extend all the components to include a `multiPageIndex` property when iterated over in `hierarchy.ts`.. :thinking: That might be better for us in the future than falling back to reading the flat layout here.

Reply:
Hehe, agree, I tried to look into it but didn't figure it out

Reply:
as in: `parent.item.baseComponentId`? That did not work for me

Reply:
Like this maybe?
`(c) => c.id === parent.baseComponentId ?? parent.id`
Looks like only group-children get a basecomponentId so if it is a non-nested group it only has id?

Reply:
I pushed a commit that adds `multiPageIndex` now - lets see if that works! :crossed_fingers:

Reply:
MVP🏅

Reply:
Works like a charm!